spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -195,14 +195,14 @@
       end
     end
 
-    describe '#kill_virtv2v' do
+    describe ':#kill_virtv2v' do
```

Comment:
Why the `:`?

Reply:
was a typo.  fixed it.

Reply:
Hey @thearifismail you don't need to mark these resolved, when you fix it github will auto mark it resolved

Reply:
It marks it as outdated, but it still shows up in the UI unless it's resolved, so the net result is a lot of clutter when trying to review a PR.